.github/workflows/main-analysis.yml

```diff
@@ -3,15 +3,61 @@ on:
   push:
     branches: [ main ]
 jobs:
+  build-tauri:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ windows-latest ]
+    runs-on: ${{ matrix.platform }}
```

Comment:
Why only windows? And if that's the case, remove the matrix and directly set `runs-on` to `windows-latest`